### PR TITLE
Created 'AddSongToPlaylist'

### PR DESCRIPTION
**Pull request by Svyeta0210**

### CODE REVIEW

File: src/test/java/Homework17.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        //Click 'Add to' the playlist button
+        WebElement addToPlaylistButton = driver.findElement(By.cssSelector("button[class='btn-add-to']"));
+        addToPlaylistButton.click();
+        //Choose the playlist 'Test Pro playlist' to add the song
+        WebElement playlistName = driver.findElement(By.xpath("//*[@id=\"songsWrapper\"]/header/div[3]/div/section[1]/ul/li[63]"));
+        playlistName.click();
+        Thread.sleep(5000);
+        //Verify that the notification message has the text 'Added 1 song into Test Pro playlist'
+        //Verify that the notification message appears at the right top of the screen
+       Assert.assertTrue(driver.findElement(By.cssSelector("div[class='alertify-logs top right']")).isDisplayed());
```
Comment (Berichicko) [inline src/test/java/Homework17.java:61]: If possible, avoid overly long methods. A long method is 10‒15 lines. If possible, you should split the method into several shorter ones or even create a new class for this.

File: src/test/java/Homework17.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+
+
```
Comment (Berichicko) [inline src/test/java/Homework17.java:64]: **Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._

File: src/test/java/Homework17.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework17 extends BaseTest{
+
+    @Test
+
+    public static void addSongToPlaylist() throws InterruptedException {
```
Comment (Berichicko) [inline src/test/java/Homework17.java:15]: **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   One test - One Assert
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}

File: src/test/java/Homework17.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    //      Added ChromeOptions argument below to fix websocket error
+    ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+
+    WebDriver driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+    String url = "https://bbb.testpro.io/";  //Navigate to "https://bbb.testpro.io/"
+        driver.get(url);
```
Comment (Berichicko) [inline src/test/java/Homework17.java:25]: Each class must have a simple purpose and, ideally, solve a single problem. This is called the single-responsibility principle (SRP).

BaseTest — Implements the necessary methods for work with webdriver.

For example

image

File: src/test/java/Homework17.java (hunk 1 above)
Comment (Svyeta0210) [reply src/test/java/Homework17.java:61]: Hello Alex, 
This is how the instructions go for our homework. There is nothing said  that we should create several test cases on this task. I had some doubts about how I did my homework but there was not much had been shown, during the class and in LMS, of how we should create those @Before and @After methods so I didn't quite get it how to do that.
Screenshot 2023-04-23 175834
Now after submitting this H/W ,I see that, what I should change in my code. Thank you for providing this information about corrections to be made.

**Pull request merged**